tasks/main.yml

```diff
@@ -25,7 +25,7 @@
           name: pgweb
           namespace: "{{ namespace }}"
           labels:
-            app: thoth
+            app: thoth-postgres
```

Comment:
shouldnt we keep all the things thoth needs within the thoth app?

Reply:
+1, adjusted